integrations/viem-v2/package.json

```diff
@@ -60,6 +60,8 @@
 	"devDependencies": {
 		"@biomejs/biome": "^1.7.0",
 		"@types/node": "^18.19.31",
+		"@zaeny/literate": "^1.0.8",
```

Comment:
Maybe a license issue

Reply:
You're right. It's silly to use an external package when it's basically just a single regex. Have rewritten to avoid the external package.